logging/otel-collector.yaml

```diff
@@ -82,17 +82,11 @@ spec:
               - set(attributes["message"], body)
 
     exporters:
-      loki:
```

Comment:
Could you please keep it here and just disable it (don't include it in the pipeline)?